libvmi/accessors.c

```diff
@@ -63,6 +63,51 @@ vmi_get_ostype(
         vmi->os_type;
 }
 
+bool
+vmi_get_windows_build_info(
+    vmi_instance_t vmi, win_build_info_t* info)
+{
+    if (!info) {
```

Comment:
This should be under ENABLE_SAFETY_CHECKS

Reply:
@tklengyel, done!